.ldrelease/update-bitbucket.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Nit: Perhaps renaming this to update-bitbucket-metadata.sh can make the intent of this file clearer?